packages/r/rsm-mmio/xmake.lua

```diff
@@ -7,27 +7,24 @@ package("rsm-mmio")
              "https://github.com/Ryan-rsm-McKenzie/mmio.git")
     add_versions("2.0.0", "360dddf74a97bd0a7eb41378cc59f2a69871dabfd36c55bf027429ac54930d5b")
 
+    add_configs("shared", {description = "Build shared library.", default = false, type = "boolean", readonly = true})
+
     on_install(function (package)
         io.writefile("xmake.lua", [[
             add_rules("mode.debug", "mode.release")
             set_languages("c++17")
             target("rsm-mmio")
-                set_kind("$(kind)")
+                set_kind("static")
                 add_files("src/**.cpp")
                 add_includedirs("include/")
                 add_headerfiles("include/(**.hpp)")
-                if is_plat("windows") and is_kind("shared") then
-                    add_rules("utils.symbols.export_all", {export_classes = true})
-                end
```

Comment:
But we can make it support shared, if it can be compiled well. Why not support it?